### PR TITLE
update change_log

### PR DESCRIPTION
**Pull request by Wzb123456789**

### CODE REVIEW

File: tools/azure-sdk-tools/packaging_tools/change_log.py [hunk 1]
```diff
@@ -67,7 +67,8 @@ def operation(self, diff_entry):
             return
 
         # Is this a new operation, inside a known operation group?
-        function_name, *remaining_path = remaining_path
+        function_name, change_parameter_name, *remaining_path = remaining_path
+        print(remaining_path)
```
Comment (msyyc) [inline tools/azure-sdk-tools/packaging_tools/change_log.py:71]: do not add surplus code

File: tools/azure-sdk-tools/packaging_tools/change_log.py [hunk 2]
```diff
@@ -67,7 +67,7 @@ def operation(self, diff_entry):
             return
 
         # Is this a new operation, inside a known operation group?
-        function_name, *remaining_path = remaining_path
+        function_name, change_parameter_name, *remaining_path = remaining_path
```
Comment (msyyc) [inline tools/azure-sdk-tools/packaging_tools/change_log.py:70]:
```suggestion
        function_name, parameter_name, *remaining_path = remaining_path
```

File: tools/azure-sdk-tools/packaging_tools/change_log.py [hunk 3]
```diff
@@ -80,7 +80,7 @@ def operation(self, diff_entry):
             return
 
         # So method signaure changed. Be vague for now
-        self.breaking_changes.append(_SIGNATURE_CHANGE.format(operation_name, function_name))
+        self.breaking_changes.append(_SIGNATURE_CHANGE.format(operation_name, function_name, change_parameter_name))
```
Comment (msyyc) [inline tools/azure-sdk-tools/packaging_tools/change_log.py:83]:
```suggestion
        self.breaking_changes.append(_SIGNATURE_CHANGE.format(operation_name, function_name, parameter_name))
```

**Review by msyyc — APPROVED** [review]

**Comment by Wzb123456789** [comment]
https://github.com/Azure/azure-sdk-for-python/issues/23480

**Pull request merged**